lib/v4/migration-helpers/update-api-folder-structure.js

```diff
@@ -121,18 +116,21 @@ const updateApiFolderStructure = async (appPath) => {
     await updateRoutes(apiPath, apiName);
     await updatePolicies(apiPath);
     // Update services using jscodeshift transform
-    runJsCodeshift(
-      join(apiDirCopyPath, apiName, "services"),
-      "use-arrow-function-for-service-export"
+    runJscodeshift(
+      join(apiDirCopyPath, apiName, 'services'),
+      'use-arrow-function-for-service-export'
     );
   }
-
-  console.log(`migrated ${basename(strapiAppPath)}/api to Strapi v4`);
-  console.log(`to see changes: Run "git add . && git diff --cached"`);
-  console.log(
-    'to revert: "git reset HEAD --hard && git cleanEmptyDirectories -fd"'
+  logger.info(`migrated ${chalk.yellow(basename(strapiAppPath) + '/api')} to Strapi v4 🚀`);
+  logger.info(
+    `to see changes: Run ${chalk.green('git')} add . && ${chalk.green('git')} diff --cached`
```

Comment:
I'd recommend putting the entire command in color, so that it's obvious what part the user has to copy and paste